geth/jail/internal/loop/loop.go

```diff
@@ -40,6 +42,27 @@ type Loop struct {
 	lock  sync.RWMutex
 	tasks map[int64]Task
 	ready chan Task
+	state
+}
+
+// state encapsulates the loop's open and closedness. This is a boolean variable
+// indicating whether the loop is open for new tasks and a lock to control
+// access to this boolean.
+type state struct {
+	accept bool
+	lock   sync.RWMutex
+}
+
+func (s *state) accepting() bool {
```

Comment:
I know I'm a bit pedantic. But I also love small comments on private functions like this one. Is simply teaching a style. Also I like naming like `isAsccepting()` or `doesAccept()` here.